docs/scripting.md

```diff
@@ -94,7 +94,7 @@ the response object a private message can be sent to the original sender.
 ```coffeescript
   robot.respond /I don't like Sam-I-am/i, (res) ->
```

Comment:
Can we also change the capitalization of `Sam-I-Am` in this section? It appears in several places below.

![9780525579588_p0_v1_s550x406](https://user-images.githubusercontent.com/80459/47223548-26cbd500-d37f-11e8-9b51-fb2ae1039bbf.jpg)

Reply:
Done.
I also lower-cased the matching text for "Sam-I-Am", since it's case insensitive.